src/main/java/frc/robot/scoreStates/ScoreStateConstants.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.scoreStates;
+
+import com.pathplanner.lib.util.GeometryUtil;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.Measure;
+import edu.wpi.first.units.Units;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ScoreStateConstants {
```

Comment:
It'll be more organized to store each constant in it's respective state manager.

Reply:
Do you mean put them in the state files or create a separate constants class for each state

Reply:
In the state files. Usually I like to put constants in the relevant class, especially if the class itself isn't too long.